var/spack/repos/builtin/packages/saga-gis/package.py

```diff
@@ -38,31 +39,52 @@ class SagaGis(AutotoolsPackage):
 
     variant('gui',      default=True,   description='Build GUI and interactive SAGA tools')
     variant('odbc',     default=True,   description='Build with ODBC support')
+
     # FIXME Saga-gis configure file disables triangle even if
     # --enable-triangle flag is used
     # variant('triangle', default=True,   description='Build with triangle.c
+
     # non free for commercial use otherwise use qhull')
-    variant('libfire',  default=True,   description='Build with libfire (non free for commercial usage)')
-    variant('openmp',   default=True,   description='Build with OpenMP enabled')
-    variant('python',   default=False,  description='Build Python extension')
+    variant('libfire',      default=True,   description='Build with libfire (non free for commercial usage)')
+    variant('openmp',       default=True,   description='Build with OpenMP enabled')
+    variant('python',       default=False,  description='Build Python extension')
+
+    variant('postgresql',   default=False,  description='Build with PostgreSQL library')
+    variant('opencv',       default=False,  description='Build with libraries using OpenCV')
 
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool',  type='build')
     depends_on('m4',       type='build')
 
-    # FIXME unnecessary dependency on python3 because of implicit python3
-    # dependency through meson by a dependency of wx/gtkplus
-    depends_on('python@3:')
```

Comment:
Is this no longer the case?

Reply:
I only added this dependency in the first place so that the default build without explicit dependencies works. When spack defaulted to python2 _in the old days_, there was a version clash of dependencies built with python2 respectively python3. Since spack defaults to python3 this is not a problem anymore.